README.md

```diff
@@ -1,20 +1,37 @@
-# cmc-ccd-submit-domain
+# CMC CCD Domain and Definitions
 
 Repo for shared CMC/CCD Java model and CMC/CCD Definitions. It will contain:
 
 * CMC CCD model
-* CMC CCD JSON Definition (*note:* see definition/README.md for how to handle defintions)
+* CMC CCD JSON Definition (*note:* [definition/README.md](./definition/README.md) for how to handle defintions)
 * Tooling to release/manage CCD Definition
 * When merging into master:
     * Release the Docker CCD Definition image
     * Release new JAR
     * Exercise cmc-claim-store and cmc-claim-submit-api functional tests (and any other consumers)
 
-## Master
+## Release
 
-Currently any merge/commit to master will trigger an ACR task to build CCD Definition Docker image. See /definitions for more details.
+Version number is defined in file [VERSION.yaml](./VERSION.yaml), both artefacts use same version number.
 
-## Development
+### Domain (Java Jar)
+
+On any tagged commit Travis will build and push to bintray: http://dl.bintray.com/hmcts/hmcts-maven/uk/gov/hmcts/reform/cmc/ccd-domain/
+
+#### To release:
```

Comment:
what about create a release in git hub?
It is generally what I do. however the tag need to be copied over.